.github/workflows/publish.yml

```
@@ -0,0 +1,62 @@
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  source-checksum:
+    permissions:
+      contents: write
+      actions: read
+    uses: ./.github/workflows/source-checksum.yml
```

actionlint (1.7.8)

could not read reusable workflow file for "./.github/workflows/source-checksum.yml": open /home/jailuser/git/.github/workflows/source-checksum.yml: no such file or directory

(expression)